__tests__/app.test.js

```diff
@@ -65,23 +65,70 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments[0]
+
+                expect(result).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number)
+                })
```

Comment:
This would be a better test if you checked that all the comments objects are correct, you can still use toMatchObject - but use it on each object by going through the array